mount/mount_linux.go

```diff
@@ -347,7 +370,12 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 		klog.Infof("Disk %q appears to be unformatted, attempting to format as type: %q with options: %v", source, fstype, args)
 		output, err := mounter.Exec.Command("mkfs."+fstype, args...).CombinedOutput()
 		if err != nil {
-			detailedErr := fmt.Sprintf("format of disk %q failed: type:(%q) target:(%q) options:(%q) errcode:(%v) output:(%v) ", source, fstype, target, options, err, string(output))
+			// Do not log sensitiveOptions only options
+			sensitiveOptionsLog := ""
+			if len(sensitiveOptions) > 0 {
+				sensitiveOptionsLog = sensitiveOptionsRemoved
```

Comment:
suggest `strings.Repeat("<masked> ", len(sensitiveOptions))` so we can see how many options were masked

Reply:
or making a `optionsForLogging` slice that starts with `options` and appends `<masked>` the right number of times, so you can include a single %q in the log message and get a single list (instead of `["foo"]"<masked>"`)

Reply:
Will do.